app/src/main/java/com/github/geohunt/app/ui/components/tutorial/TutorialSlides.kt

```diff
@@ -0,0 +1,38 @@
+package com.github.geohunt.app.ui.components.tutorial
+
+import com.github.geohunt.app.R
+
+/**
+ * The data class for the tutorial slides
+ *
+ * @param icon The icon displayed on the slide
+ * @param title The title of the slide
+ * @param description The description for the slide
+ */
+class TutorialSlides(
+    val icon: Int,
+    val title: Int,
+    val description: Int
+) {
+    companion object{
+        fun getData(): List<TutorialSlides>{
+            return listOf(
+                TutorialSlides(
+                    R.drawable.tutorial_icon_1,
```

Comment:
You should give clearer names to the string ressources, first_screen, icon_1 etc aren't really helpful.

Reply:
I have changed the names of the tutorial_icons so that they all follow the same style. I think that writing the names for resources of the form  `tutorial_(screen index)_screen_(icon name)` is very clear name for a string resource.